### DISCUSSION TITLE
Seq is unavailable - Windows OS Server

### DISCUSSION THREAD

**Discussion by Kingston-Wharves** [datalust/seq-tickets#2334] [Help]
Receiving the below error. Let me know what other details should be provided to assist in troubleshooting.

{"Error":"Seq is unavailable. Failed to initialize storage: Flare native storage failed (InternalError), data file `\"C:\\\\ProgramData\\\\Seq\\\\Extents\\\\2023-10-04_2023-10-11\\\\extent.08dbc790d113a200_08dbc859fb7d6200.9cf09d05adbb4b2a9beede044bae5ff1.span\"` is missing."}

**Comment by nblumhardt** [comment]
Hi! It looks like you could be running a fairly old/outdated Seq version; the quick fix, if you're not concerned about losing data, will be to stop the Seq service, delete the `2023-10-04_2023-10-11` folder mentioned in the error message, and start back up. You'll lose all stored event data in this time range.

If the Seq version is new recent enough to have `flaretl.exe` bundled with it, stopping the service and running:

```
flaretl.exe repair --metadata "C:\ProgramData\Seq\Extents\2023-10-04_2023-10-11\extent.flare"
```

might work, and preserve more of the data in the corrupted time range.

If you have a supported Seq subscription, please shoot as many details as you can about the Seq installation (run `seq version` at the command-line, let us know what OS it's running on, machine specs, amount of stored data) through to support@datalust.co, and we'll do our best to walk you through the process, and get you back up and running on a more recent version.

Hope this helps,
Nick

**Comment by Kingston-Wharves** [comment]
Hi Nick,

I tried what you suggested. Well, I didn't delete the folder but renamed it. What happened is that the error message remained but was adjusted to reference a different folder name. I'll take your advice and send a message through to support. 

Thanks again
KWL

**Comment by Kingston-Wharves** [comment]
Hi Nick,

I did some further digging, our archiving process moved the referenced log folders to a different drive. When I copied them back to the configured SEQ Storage Path, the service came back online. Since we need to maintain our archiving process for our servers, how can I properly archive these log folders without interrupting SEQ service?

SEQ Version 5.1.3200 (yes very outdated I am now discovering)
Windows Server 2016

~KWL

**Reply by nblumhardt** [reply]
Hi! Seq supports retention policies for removing old data; see https://docs.datalust.co/docs/retention-policies.

Hope this helps,
Nick
